### ISSUE TITLE
Cannot clone cobbler repo to Windows

### ISSUE THREAD

**Issue by movermeyer** [cobbler/cobbler#1722]
If you try to clone the cobbler repository to a Windows machine, you get the following error:

```
git.exe clone --progress -v "https://github.com/cobbler/cobbler.git" "D:\cobbler"

Cloning into 'D:\cobbler'...
POST git-upload-pack (gzip 4901 to 2499 bytes)
remote: Counting objects: 40362, done.
remote: Compressing objects: 100% (39/39), done.
remote: Total 40362 (delta 16), reused 0 (delta 0), pack-reused 40323
Receiving objects: 100% (40362/40362), 16.69 MiB | 2.22 MiB/s, done.
Resolving deltas: 100% (27284/27284), done.
fatal: cannot create directory at 'aux': Invalid argument
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'

git did not exit cleanly (exit code 128) (9328 ms @ 2016-11-21 16:29:39)

```

It seems that 'aux' is a reserved word in Windows and therefore you cannot check out https://github.com/cobbler/cobbler/tree/master/aux on Windows.

A quick Google search lead me to [this issue in another project](https://github.com/swcarpentry/DEPRECATED-bc/issues/463). They ended up switching `aux` to `misc`

**Comment by jmaas** [comment]
you should be able to checkout the code now.

the first release with this fixed will be 2.8.1

**Comment by movermeyer** [comment]
I tried this again and found that there is actually another `aux` directory: [`tests/aux`](https://github.com/cobbler/cobbler/tree/1d582e746bf6f5e907d0080c393e6a5d32874eb3/tests/aux)
